### PR TITLE
plot gages without percentiles

### PR DESCRIPTION
**Pull request by wdwatkins**
Adding shape and size to dv_stat_colors now.  Had to shuffle where the config options get passed in in a few places.

I didn't commit any build/indicator files here, since I've just been running this for a few days to test.

### CODE REVIEW

File: 2_process/src/process_dv_stat_colors.R [hunk 1]
```diff
@@ -3,15 +3,20 @@
...
   dv_stats_with_color <- dv_stats %>%
-    filter(!is.na(per)) %>%
-    mutate(color = rgb(col_fun(per), maxColorValue = 255)) # don't know how necessary maxColorValue is
+    mutate(shape = ifelse(is.na(per), no = gage_style$with_percentile$pch,
+                         yes = gage_style$no_percentile$pch),
+          size = ifelse(is.na(per), no = gage_style$with_percentile$cex,
+                        yes = gage_style$no_percentile$cex),
+          color = NA)
+  per_na <- is.na(dv_stats$per)
+  #this was not working with ifelse inside mutate, still NAs getting into rgb/col_fun
```
Comment (lindsaycarr) [inline 2_process/src/process_dv_stat_colors.R]: I wonder if you needed to do `rowwise()` or something?

**Comment by aappling-usgs** [comment]
Just checking - if you've been running this to test and without plans to commit .ind or build/status files, you've been using `options(scipiper.dry_put=TRUE)`, right?

File: 2_process/src/process_dv_stat_colors.R (hunk 1 above)
Comment (wdwatkins) [reply 2_process/src/process_dv_stat_colors.R]: Using `rowwise` made it not error, but the result was still wrong 😕

Comment (lindsaycarr) [reply 2_process/src/process_dv_stat_colors.R]: well this works with your base R solution below, so moving on!

**Comment by lindsaycarr** [comment]
Fixes #25

**Pull request merged**